packages/neovim-git/neovim-git.pacscript

```diff
@@ -1,21 +1,85 @@
 pkgname="neovim-git"
 gives="neovim"
-source=("https://github.com/neovim/neovim.git")
-pkgver="0.9.1"
+pkgver="0.10.1"
+repology=("project: ${gives}")
+source=(
+  "https://github.com/neovim/neovim.git"
+  "ex"
+  "rview"
+  "rvim"
+  "view"
+  "vimdiff"
+)
+sha256sums=(
+  "SKIP"
+  "5d3178628afc39f0318638cad19e9ad06ada89926dfb263e93ab35cc8f60e911"
+  "f04c9baa97f49afe7969826b45e283c60e2d52e28aa3ea7fa93a62dab0408a55"
+  "8528f61eaca19285c8220c7d12099743d76abf872a30515698644c91a05c55e0"
+  "3cb963a264b6cf45749627408b5a62c6945e5e426b595bb5e6a388194c2c1990"
+  "1d3123b9d97a4889696ab72db558041b5cca8e26ff51ae557f7842242a7d4ed1"
+)
 url='https://neovim.io/'
 pkgdesc="Vim fork maintained by the community"
 maintainer=("Elsie19 <elsie19@pm.me>")
 makedepends=("ninja-build" "gettext" "libtool-bin" "autoconf" "automake" "cmake" "g++" "pkg-config" "unzip")
+replaces=("neovim" "nvim" "neovim-runtime" "neovim-git" "neovim-app" "neovim-deb" "neovim-bin")
+conflicts=("${replaces[@]}")
 external_connection=true
-breaks=("neovim-bin" "neovim" "neovim-app" "neovim-deb")
 
 build() {
-  cd "${_archive}"
+  cd "${gives}"
   make CMAKE_BUILD_TYPE=Release CMAKE_INSTALL_PREFIX=/usr -j"${NCPU}"
 }
 
 package() {
-  cd "${_archive}"
+  cd "${gives}"
   make install DESTDIR="${pkgdir}"
+  install -Dm644 "LICENSE.txt" -t "${pkgdir}/usr/share/licenses/${gives}"
+  install -Dm644 "README.md" -t "${pkgdir}/usr/share/doc/${gives}"
+  install -Dm755 "${srcdir}/ex" -t "${pkgdir}/usr/libexec/${gives}"
+  install -Dm755 "${srcdir}/rview" -t "${pkgdir}/usr/libexec/${gives}"
+  install -Dm755 "${srcdir}/rvim" -t "${pkgdir}/usr/libexec/${gives}"
+  install -Dm755 "${srcdir}/view" -t "${pkgdir}/usr/libexec/${gives}"
+  install -Dm755 "${srcdir}/vimdiff" -t "${pkgdir}/usr/libexec/${gives}"
 }
+
+post_install() {
+  set -e
+
```

Comment:
```suggestion
```

Reply:
And why would you do this? This is actually intentionally set in this specific case.

Reply:
Because I can't think of a situation where this would ever fail unless something way more catestrophic situation happened.

Reply:
The intent is to literally report on ANY issues in the post-installation setup, because it's actually making system changes, this is critical. Without this, it would only report if the *last* command run were to exit with a non-zero return code. set -e, in this, insures it stops early, and reports post installation, and pre-removal issues that effect system changes. And it's also been in the Debian neovim package for 8 years and counting, for this reason.

Reply:
https://salsa.debian.org/vim-team/neovim/-/commit/db8850a951b1eb8085a2b182515b2f0884f105c1 -- For reverence.